e2e/network.go

```diff
@@ -0,0 +1,158 @@
+package e2e_test
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/Azure/agentbakere2e/suite"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork"
+)
+
+var (
+	nsgName           string = "abe2e-airgap-securityGroup"
+	defaultSubnetName string = "aks-subnet"
+)
+
+func cloudGapSecurityGroup(location, kubernetesEndpont string) armnetwork.SecurityGroup {
```

Comment:
cloudGap or airGap?

Reply:
airgap - definitely was blind to this name since it was the first function I started with lol